src/python/grapl_e2e_tests/tests.py

```diff
@@ -57,6 +62,27 @@ def scope_has_N_items() -> bool:
             timeout_secs=40,
         )
 
+    # -------------------------- MODEL PLUGIN TESTS -------------------------------------------
+
+    def test_upload_plugin(self) -> None:
+        upload_model_plugin(model_plugin_client)
+
+    @pytest.mark.xfail  # TODO: Remove once list plugins is resolved
+    def test_list_plugin(self) -> None:
+        get_plugin_list(model_plugin_client)
+
+    @pytest.mark.xfail  # TODO: once list plugins is resolved, we can fix delete plugins :)
+    def test_delete_plugin(self) -> None:
+        delete_model_plugin(model_plugin_client, "schemas.py")  # Hard Code for now
+        gql_client = GraphqlEndpointClient(jwt=EngagementEdgeClient().get_jwt())
+
+        wait_for_one(
```

Comment:
Could we also add a check here that the `graphql lens scope` includes the new schemas uploaded?
(Sidenote: I suspect they *do not* - I may not have added the "provision again" code to the model plugin deployer)

Reply:
Addressing here; https://github.com/grapl-security/issue-tracker/issues/395